tests/test_manage_py_scan.py

```diff
@@ -118,7 +118,13 @@ def test_django_project_found_invalid_settings_version(django_testdir, monkeypat
 
     result = django_testdir.runpytest_subprocess("django_project_root", "--version", "--version")
     assert result.ret == 0
-    result.stderr.fnmatch_lines(["*This is pytest version*"])
+    if result.errlines:
+        version_out = result.stderr
+    else:
+        # Pytest 7.0+
+        version_out = result.stdout
+
+    version_out.fnmatch_lines(["*This is pytest version*"])
```

Comment:
I prefer to use an explicit pytest version check, since then when we drop support for pytest<7 it is much easier to find and remove no longer needed code.

```suggestion
    if hasattr(pytest, 'version_info') and pytest.version_info >= (7, 0):
        version_out = result.stdout
    else:
        version_out = result.stderr
    version_out.fnmatch_lines(["*This is pytest version*"])
```

Reply:
Yes, this is better! Thanks @bluetech

Reply:
@bluetech, thank you for the suggestion! I thought about this.
I prefer to not rely on version-check but check for actual behavior. For future cleanup I added the comment in code. But of course, your pov as project's maintainer is higher than my own.

Reply:
updated.
Note: `version_info` => `version_tuple` (requires Pytest 7.0+).